accounting/bank/feeds/bank_synchronization.rst

```diff
@@ -109,23 +182,28 @@ until the security code is provided.
 Not all of my past transactions are in Odoo, why?
 -------------------------------------------------
 
-Transactions can only be fetched up to 3 months in the past.
+For some institutions, transactions can only be fetched up to 3 months in the past.
 
-Why don’t I see any transactions?
+Why don't I see any transactions?
 ---------------------------------
-When you first connect with your bank, you will be prompted to add each account to its own journal.
-If you skip this step, you will not be able to see your transactions in Odoo.
+
+During your first synchronization, you selected the bank accounts you decided to synchronize with
+Odoo. If you didn't synchronize any of your accounts, you can go to :menuselection:`Accounting -->
+Configuration --> Online Synchronization` to click on the *Fetch Accounts* button on the connection.
+
+There may also be no new transactions.
 
 If your bank account is properly linked to a journal and posted transactions are not visible in your
 database, please `submit a support ticket <https://www.odoo.com/help>`_.
 
 How can I update my bank credentials?
 -------------------------------------
-You can update your credentials in
-:doc:`Developer mode <../../../general/developer_mode/activate>`.
 
-Then go to :menuselection:`Accounting --> Configuration --> Online Synchronization`, and open the
-Institution you want to edit, and click on *Update Credentials*.
+You can update your credentials by going to :menuselection:`Accounting --> Configuration -->
+Online Synchronization`, open the connection you want to update your credentials and click on
+the *Update Credentials* button.
 
 .. seealso::
    * :doc:`bank_statements`
+   * :doc:`ponto`
+   * :doc:`saltedge`
```

Comment:
this line should be in the commit adding saltedge, otherwise this commit has a broken link.